src/components/pages/Home/Onboarding/Steps/DownloadMetamask.tsx

```diff
@@ -31,17 +33,29 @@ export default function DownloadMetamask(): ReactElement {
   const { title, subtitle, body, image, buttonLabel }: OnboardingStep =
     data.file.childStepsJson
 
-  const downloadMetamask = () =>
-    window.open(
-      'https://metamask.io/download/',
-      '_blank',
-      'noopener noreferrer'
-    )
+  const onboarding = useRef<MetaMaskOnboarding>()
+
+  useEffect(() => {
+    if (!onboarding.current) {
+      onboarding.current = new MetaMaskOnboarding()
+    }
+  }, [])
```

Comment:
issue(non-blocking): we have some duplication here with the account button. It seems like it would be a good idea to refactor this into its own hook to be usable in different components.
Marked as `non-blocking` for now to be able to proceed with the fix, but should be tracked as an issue to be tackled whenever we have the time.